### ISSUE TITLE
Not compatible with Postal addon

### ISSUE THREAD

**Issue by Anothersignup** [GoldpawsStuff/ChatCleaner#9]
Postal addon will no longer show how much money I gained from Auction House, and which items was sold.

**Comment by Goldpaw** [comment]
chat cleaner queues up money events when certain windows like the mailbox, auction house or merchants are open. And then shows the gain/loss as simplified messages after. 

this is part of the money module, and that's currently not optional as it works a bit different from the others. I can add an option to turn off money filtering, but I have no ETA on this. :o

**Comment by Starlynk1** [comment]
Started using Postal a while ago and didn't have any issues with the money.lua component. After the recent update, I am not getting errors with the money.lua file and more specifically, line 327.

```16x ChatCleaner/Components/Money.lua:327: Usage: HookScript(object, method, [handler]): You can only hook a script on a frame object
[string "=[C]"]: ?
[string "@Bartender4/libs/AceHook-3.0-9/AceHook-3.0.lua"]:143: in function <...eBartender4/libs/AceHook-3.0/AceHook-3.0.lua:118>
[string "@Bartender4/libs/AceHook-3.0-9/AceHook-3.0.lua"]:366: in function `HookScript'
[string "@ChatCleaner/Components/Money.lua"]:327: in function `?'
[string "@ChatCleaner/Libs/LibMoreEvents-1.0-4/LibMoreEvents-1.0.lua"]:91: in function <...Cleaner/Libs/LibMoreEvents-1.0/LibMoreEvents-1.0.lua:87>
[string "=(tail call)"]: ?

Locals:
(*temporary) = "Usage: HookScript(object, method, [handler]): You can only hook a script on a frame object"
```

**Comment by Goldpaw** [comment]
Ah, yes, I can see the reason for that problem. 'ClassTrainerFrame' isn't always loaded, and everywhere except for that line the addon checks for it. 

Fix for that coming in 2.0.43-Release, which should be available shortly.
